libvast/vast/system/source.hpp

```diff
@@ -77,47 +77,54 @@ caf::behavior
 source(caf::stateful_actor<source_state<Reader>>* self, Reader&& reader) {
   using namespace caf;
   using namespace std::chrono;
+  // Initialize state.
   self->state.reader = std::move(reader);
   self->state.name = self->state.reader.name();
-  auto eu = self->system().dummy_execution_unit();
-  self->state.sink = actor_pool::make(eu, actor_pool::round_robin());
+  // Fetch accountant from the registry.
   if (auto acc = self->system().registry().get(accountant_atom::value))
     self->state.accountant = actor_cast<accountant_type>(acc);
+  // We link to the importers and fail for the same reason, but still report to
+  // the accountant.
   self->set_exit_handler(
     [=](const exit_msg& msg) {
-      if (self->state.accountant) {
+      auto& st = self->state;
```

Comment:
The extra `st` doesn't strike me worth the extra line in this case. (In other cases, it surely helps readability.)